src/api/tests/test_validations.py

```
@@ -232,3 +233,240 @@ def test_habcomp_protocol_validation_ok(valid_habitat_complexity_collect_record,
+    assert overall_status == ERROR
+    assert _get_result_status(results["$record"], "duplicate_validator") == ERROR
+
+
+def test_belt_invert_density_per_goi_warn(
+    valid_belt_invert_collect_record,
+    profile1_request,
+    invert_species_1,
+    invert_belt_transect_width_1m,
+):
+    # invert_species_1 links to invert_genus_1 → invert_group_of_interest_1 (sea urchins, 5500/ha).
+    # 1m wide × 50m long = 50 m²; need > 5500/ha → >27.5 individuals → 28 individuals.
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)